### ISSUE TITLE
Accessibility - cannot access close button on miniplayer if not hovored over

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#7036]
"The one problem I have is that the close button is removed from the DOM when I'm not hovering over it with my mouse instead of being just hidden,
I'd like to click it with my keyboard so it needs the button to target."

**Comment by nath1as** [comment]
I checked and it's not actually removed from DOM, its wrapped in a way that vimium plugin (and possibly other extensions for keyboard browsing) can't detect it.
